backend/de.metas.manufacturing.rest-api/src/main/java/de/metas/handlingunits/rest_api/move_hu/MoveHUCommand.java

```diff
@@ -138,4 +166,39 @@ private void moveHUsToLocator(final @NonNull List<I_M_HU> husToMove, @NonNull fi
 																					.movementDate(SystemTime.asInstant())
 																					.build()));
 	}
+
+	@NonNull
+	private Optional<HuPackingInstructionsItemId> getNewLUPackingInstructionsForAggregateSplit(@NonNull final HuId tuId)
+	{
+		if (!placeAggHusOnNewLUsWhenMoving)
+		{
+			return Optional.empty();
+		}
+
+		final I_M_HU_PI tuPI = handlingUnitsBL.getEffectivePI(tuId);
+		if (tuPI == null)
+		{
+			return Optional.empty();
+		}
+
+		final I_M_HU_PI_Item luItem = handlingUnitsDAO.retrieveDefaultParentPIItem(tuPI, X_M_HU_PI_Version.HU_UNITTYPE_LoadLogistiqueUnit, null);
```

Comment:
wondering why not introducing 
Optional<HuPackingInstructionsItemId> IHandlingUnitsDAO.retrieveDefaultParentPIItemId(...)